### PR TITLE
[4.1] A new Icon for com_tags to the quickicon module

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue # .
Suggested by a user in the german community.

### Summary of Changes
This PR adds a new icon for com_tags to the Quickicon panel site. For users with many tags this is very useful.

### Testing Instructions
Edit the module quickicon site. 
Activate the new icon tags, if you want with counting items.

quickicon-tags1

Then you should see the Icon in your dashboard. Make sure that ony published tags are counted 

quickicon-tags2

### Documentation Changes Required
yes

### CODE REVIEW

File: administrator/language/en-GB/com_tags.ini [hunk 1]
```diff
@@ -92,6 +92,12 @@ COM_TAGS_N_ITEMS_TRASHED="%d tags trashed."
 COM_TAGS_N_ITEMS_TRASHED_1="Tag trashed."
 COM_TAGS_N_ITEMS_UNPUBLISHED="%d tags unpublished."
 COM_TAGS_N_ITEMS_UNPUBLISHED_1="Tag unpublished."
+COM_TAGS_N_QUICKICON="Tags"
+COM_TAGS_N_QUICKICON_0="Tag"
```
Comment (brianteeman) [inline administrator/language/en-GB/com_tags.ini]:
```suggestion
COM_TAGS_N_QUICKICON_0="Tags"
```

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on a5265d6e52a3e2809522a430664a30cbdfae44dd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32223">issues.joomla.org/tracker/joomla-cms/32223</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on a5265d6e52a3e2809522a430664a30cbdfae44dd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32223">issues.joomla.org/tracker/joomla-cms/32223</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32223">issues.joomla.org/tracker/joomla-cms/32223</a>.</sub>

**Comment by infograf768** [comment]
Tks

**Comment by infograf768** [comment]
@wilsonge 

imho this could have gone into 4.0

**Comment by PhilETaylor** [comment]
[4.1-dev] 0 Access to undeclared static property: Joomla\Module\Quickicon\Administrator\Helper\QuickIconHelper::$buttons #35727

**Pull request merged**